### ISSUE TITLE
Too many issues 

### ISSUE THREAD

**Issue by lastkingx92** [iabem97/saigon#132]
Hey . I was able to jailbreak my iphone 5s using your saïgon beta 3 but unfortunately I am facing many problems with my iPhone. 
1) When i reboot my device cydia icon is available there but when I click on the icon it doesn't open & then when I open saïgon app it tells me to jailbreak . 
2) After jailbreaking again. This time cydia works but when I try to install any tweak it shows DPKG_Locked (or sometimes it downloads the tweak but doesn't show [Return to cydia or Respring Option and keeps on loading]) .
3) The tweaks that were downloaded before are also not showing on the home screen. 
Please help me I am in too much trouble right now.

**Comment by xevidos** [comment]
In response to your questions,
1. This is a semi-untethered jailbreak:
"The semi-untethered jailbreak requires you to re-run the jailbreak tool every time you reboot your device, but it doesn't require the use of a computer. It's much more convenient than a tethered jailbreak, but not quite as convenient as an untethered jailbreak."

2. There are many different tutorials you should try to see if they fix your issues.  There are text tutorials or even videos if you are more of a video person. [This is the google page.](https://www.google.com/search?q=fix+dpkg+locked+cydia+IOS+10&oq=fix+dpkg+locked+cydia+IOS+10&aqs=chrome..69i57.6788j0j9&sourceid=chrome&ie=UTF-8)

3.  Some tweaks will not show up on your homescreen depending on what type of tweak they are.  Some will show up in your settings but only when you are jailbroken.  Meaning, after you reboot you wont see them in your settings until you re-jailbreak.

**Comment by lastkingx92** [comment]
Thanks for replying! But tweaks like filza/ifile are ment to be shown on home screen but they are not shown because whenever I try to install any tweak from cydia . Cydia must show Respring or return to cydia icon . But my mobile always stuck on loading and nothing happens .

**Comment by lastkingx92** [comment]
And can i reset my iphone from settings as this semi-untethered jailbreak  ? Because once I tried to do that on 9.x but my iphone stuck on apple logo . And then I restored my iphone which caused me to update using pc.

**Comment by lastkingx92** [comment]
👆on ios 9.x i was jailbroken

**Comment by xevidos** [comment]
No matter what kind of jailbreak, you should never restore through settings.  Also, the dpkg error could be causing the packages to fail on install.  Maybe try to reinstall the tweaks after you fix the dpkg error.

**Comment by lastkingx92** [comment]
The saddest thing is I don't have a pc 😭

**Comment by xevidos** [comment]
You could use Cydia Eraser.  You just can not restore through settings.

**Comment by lastkingx92** [comment]
I can't install anything or if anything get installed it doesn't show on homescreen ! I am screwed 😰

**Comment by lastkingx92** [comment]
I had tried to install cydia eraser but it was not shown up

**Comment by xevidos** [comment]
Do you get any errors when refreshing your sources?

**Comment by lastkingx92** [comment]
Sometimes DPKG_Locked and from yesterday apt - get update. 
Loading doesn't stops

**Comment by lastkingx92** [comment]
https://www.dropbox.com/sh/g9tz2g34fllqz8p/AAD4DcE-UElC3B86lLz8YFmWa?dl=0 
This is also i get

**Comment by lastkingx92** [comment]
And everytime i open cydia it shows me to update but it is removing cydia installer and other cydia items . I am confused. Should I do continue updating or anything else you can tell . https://www.dropbox.com/s/7difl9gu28dcat8/Photo%20Dec%2014%2C%2012%2054%2036%20PM.png?dl=0 
See this picture and you will understand

**Comment by lastkingx92** [comment]
And now I have came to know that I also cannot download anything from AppStore. I am screwed 😡😡😡😡

**Comment by xevidos** [comment]
Have you tried to use cydia eraser and start over?  Can you install cydia eraser or does it still not show up?

**Comment by lastkingx92** [comment]
Yeah I had tried to install cydia eraser but it was not shown up on my home screen.

**Comment by lastkingx92** [comment]
Can i sideload cydia eraser?
